src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
-body {
+@import '../styles/card';
+@import '../styles/image';
+@import '../styles/starts';
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), when importing several files, each import statement should be on a new line and should not have a file extension. The import statements here are correctly placed on separate lines, but the file extensions are missing. Please add '.scss' to the end of each import path.